### ISSUE TITLE
loss function has problem

### ISSUE THREAD

**Issue by Seraphli** [Seraphli/gym-rl#29]
should be tf.reduce_mean(tf.square(loss))
